src/state/migrations/3.ts

```diff
@@ -28,17 +28,19 @@ export function migration3(state: PersistedState | undefined): V3State {
 
   const appUiConfigs = Object.entries(appUiConfigsLocalStorageKeys).reduce(
     (acc, [key, localStorageKey]) => {
-      const value = parseStorageItem(localStorage.getItem(localStorageKey));
+      const value = localStorageKey
+        ? parseStorageItem(localStorage.getItem(localStorageKey))
+        : undefined;
       return {
         ...acc,
         [key]: value ?? appUiConfigsInitialState[key as AppUiConfigsKeys],
       };
     },
-    {} as AppUIConfigsState
+    { favoritedMarkets: [] } as Record<AppUiConfigsKeys, any>
```

Comment:
will this work since some users were already migrated so this will be empty? What's the intent here exactly?